src/app/models/types/portraitVideoCarousel.ts

```diff
@@ -0,0 +1,23 @@
+export interface PortraitVideoItem {
+  id: string;
+  images?: Array<{
+    url: string;
+    urlTemplate?: string;
+    altText?: string;
+    type?: string;
+    source?: string;
+    height?: number;
+    width?: number;
+    orientation?: 'portrait' | 'landscape';
```

Comment:
This can only ever be two values from what i've seen (no nulls) but can be changed to string if needed, felt this was stronger